Behavioral.Automation.Bindings.UI/Abstractions/IWebElement.cs

```diff
@@ -0,0 +1,12 @@
+using Behavioral.Automation.Bindings.UI.Context;
+
+namespace Behavioral.Automation.Bindings.UI.Abstractions;
+
+public interface IWebElement
+{
+    public WebContext WebContext { get; }
+    public ElementSelector ElementSelector  { get; }
+    public string? Description { get; set; }
```

Comment:
What is the logic with nullable description? Can we still interact with an element without description in the framework?

Reply:
We can interact with the element. Description can be used to provide optional additional information to the element. We found it very convenient to use in some projects